project/Dependencies.scala

```diff
@@ -102,9 +102,9 @@ object Dependencies {
 
   val scalaXml = Def.setting(
     if (scalaBinaryVersion.value == "3") {
-      "org.scala-lang.modules" %% "scala-xml" % "2.0.1"
+      "org.scala-lang.modules" %% "scala-xml" % "2.1.0"
```

Comment:
Out of curiosity, was it intentional to keep the if statement when both branches return the same value? Or is it a mistake that both return the same?

Reply:
We can get rid of the if expression here.